components/clp-package-utils/clp_package_utils/scripts/start_clp.py

```diff
@@ -912,6 +912,9 @@ def start_webui(
         "ClientDir": str(container_webui_dir / "client"),
         "LogViewerDir": str(container_webui_dir / "yscope-log-viewer"),
         "StreamTargetUncompressedSize": container_clp_config.stream_output.target_uncompressed_size,
```

Comment:
off topic: @haiqi96 do you know why we read some of the settings from container_clp_config instead of clp_config?